website/content/docs/operations/metrics-reference.mdx

```diff
@@ -191,6 +193,7 @@ task driver; not all task drivers can provide all metrics.
 |-----------------------------------------------|-------------------------------------------------------------------|-------------|---------|--------------------------------------------------|
 | `nomad.client.allocs.complete`                | Number of complete allocations                                    | Integer     | Counter | alloc_id, host, job, namespace, task, task_group |
 | `nomad.client.allocs.cpu.allocated`           | Total CPU resources allocated by the task across all cores        | MHz         | Gauge   | alloc_id, host, job, namespace, task, task_group |
+| `nomad.client.allocs.cpu.counter_ticks`       | CPU ticks consumed by the process since startup                   | Integer     | Counter | alloc_id, host, job, namespace, task, task_group |
```

Comment:
A task can have multiple processes, so this is a little more accurate. We'll want to move this to the right spot in alphabetical order as well for the name.

```suggestion
| `nomad.client.allocs.cpu.total_ticks_count`       | Total CPU ticks consumed by the task since startup                   | Integer     | Counter | alloc_id, host, job, namespace, task, task_group |
```